library.properties

```diff
@@ -0,0 +1,11 @@
+name=Azure SDK for C
+version=1.0.0
+author=Microsoft Corporation
+maintainer=Microsoft <iotcert@microsoft.com>
+sentence=Azure SDK for C library (1.3.0-beta.2) for Arduino.
+paragraph=Arduino port of the Azure SDK for C. It allows you to use your Arduino with the Azure services like IoT Hub and Azure Device Provisioning Service.  See readme.md for more details.  Copyright (c) Microsoft. All rights reserved.  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+category=Communication
+url=https://github.com/Azure/azure-sdk-for-c
+architectures=*
+includes=az_core.h,az_iot.h,azure_ca.h
+
```

Comment:
nit question: Do other Arduino libraries have this new line/is it expected for correctness, or did we just put it there because we wanted it?

Reply:
Other Arduino libraries have an extra line in this file. Indeed an extra empty line is expected in most code files.